etcdctl/ctlv3/ctl.go

```diff
@@ -95,7 +95,8 @@ func init() {
 		command.NewAuthCommand(),
 		command.NewUserCommand(),
 		command.NewRoleCommand(),
-		command.NewCheckCommand(),
+		// Removed due to negative impact on etcd.
+		// command.NewCheckCommand(),
```

Comment:
Would it make sense to keep the command but adjust the method to output a useful message for admins.

check perf command is no longer supported in OpenShift. Performance analysis should be performed using metrics, please see the etcd dashboard.

Otherwise, I fear we will get more bugs :).

Reply:
I think thats a good suggestion yes, can do that!

Reply:
@hexfusion what do you think, or should I add a few more !! or >> << chars around it?

```
➜  etcd git:(remove-perf-cli) ✗ ./bin/etcdctl check perf --help
NAME:
	check - command no longer supported in OpenShift

USAGE:
	etcdctl check <subcommand> is no longer supported in OpenShift. Performance analysis should be performed using metrics, please see the etcd dashboards [flags]
```